src/edgeProxy/public/javascripts/clientScript.js

```diff
@@ -11,48 +23,13 @@ document.getElementById("login").addEventListener("click", () => {
 document.getElementById("logout").addEventListener("click", () => {
     // Remove "PDS_loginToken" cookie value
     document.cookie = "PDS_loginToken=; max-age=-1; path=/";
-    document.getElementById("message").innerHTML = "You have successfully logged out.";
-});
 
-// The event listener for the "get preferences" button click
-document.getElementById("get_prefs").addEventListener("click", () => {
-    fetch("/api/get_prefs").then(response => {
-        showResponse(response, "Received preferences: <br />");
-    });
-});
+    // Update UIO with preferences fetched from the unauthenticated store
+    uio.store.settingsStore.applier.change("isLoggedIn", false);
 
-// The event listener for the "save preferences" button click
-document.getElementById("save_prefs").addEventListener("click", () => {
-    const uiSettingsCookie = document.cookie
-        .split("; ")
-        .find(row => row.startsWith("fluid-ui-settings="))
-        .split("=")[1];
-
-    if (!uiSettingsCookie || uiSettingsCookie === "") {
-        document.getElementById("message").innerHTML =
-            "Note: Please click \"show preferences\" at the top right corner, select your UI preferences. Then click \"Save Preferences\" to save selected preferences to Personal Data Server.";
-    } else {
-        const uiSettings = JSON.parse(decodeURIComponent(uiSettingsCookie));
-        fetch("/api/save_prefs", {
-            method: "POST",
-            headers: {
-                "Content-Type": "application/json"
-            },
-            body: JSON.stringify(uiSettings.preferences ? uiSettings.preferences : {})
-        }).then(response => {
-            showResponse(response, "Saved preferences: <br />");
-        });
-    }
+    // Update buttons and messages on the webpage to the logged out state
+    updateLoggedInState(false);
 });
 
-const showResponse = function (response, successHeader) {
-    if (response.status === 200) {
-        response.json().then(res => {
-            document.getElementById("message").innerHTML = successHeader + JSON.stringify(res);
-        });
-    } else {
-        response.json().then(error => {
-            document.getElementById("message").innerHTML = JSON.stringify(error);
-        });
-    }
-};
+// On the page load, update buttons and messages on the webpage based on the logged in state
+updateLoggedInState(getCookieValue("PDS_loginToken") ? true : false);
```

Comment:
Might be able to write this as 
```suggestion
updateLoggedInState(!!getCookieValue("PDS_loginToken"));
```